lib/oli_web/live/products/payments/discounts_form.ex

```diff
@@ -0,0 +1,45 @@
+defmodule OliWeb.Products.Payments.DiscountsForm do
+  use Surface.Component
+
+  import Ecto.Changeset
+
+  alias Surface.Components.Form
+  alias Surface.Components.Form.{ErrorTag, Field, Label, Select, NumberInput, TextInput}
+  alias OliWeb.Common.Properties.ReadOnly
+
+  prop changeset, :changeset, required: true
+  prop discount, :any, required: true
+  prop save, :event, required: true
+  prop clear, :event, required: true
+  prop institution_name, :string
+
+  def render(assigns) do
+    ~F"""
+      <Form for={@changeset} submit={@save}>
+        <ReadOnly label="Institution" value={@institution_name}/>
+
+        <Field name={:type} class="form-group">
+          <Label />
+          <Select class="form-control" options={"Percentage": "percentage", "Fixed amount": "fixed_amount"} selected={get_field(@changeset, :type)}/>
+          <ErrorTag/>
+        </Field>
+
+        <Field name={:amount} class="form-group">
+          <Label/>
+          <TextInput class="form-control"/>
+          <ErrorTag/>
+        </Field>
+
+        <Field name={:percentage} class="form-group">
+          <Label/>
+          <NumberInput class="form-control"/>
+          <ErrorTag/>
+        </Field>
```

Comment:
Does it make sense to disable one or the other depending on the discount type? Or maybe have only one input field that stands for the corresponding discount based on type.